src/Invictus.Testing.LogicApps/LogicAppAuthentication.cs

```diff
@@ -66,6 +66,41 @@ public static LogicAppAuthentication UsingServicePrincipal(string tenantId, stri
                 () => AuthenticateLogicAppsManagementAsync(subscriptionId, tenantId, clientId, clientSecret));
         }
 
+        /// <summary>
+        /// Uses a accessToken to authenticate with Azure.
+        /// </summary>
+        /// <param name="subscriptionId">The ID that identifies the subscription on Azure.</param>
+        /// <param name="accessToken">The token to use to call the Azure management API.</param>
+        public static LogicAppAuthentication UsingAccessToken(string subscriptionId, string accessToken)
+        {
+            Guard.NotNullOrWhitespace(subscriptionId, nameof(subscriptionId));
+            Guard.NotNullOrWhitespace(accessToken, nameof(accessToken));
+
+            return new LogicAppAuthentication(
+                () => AuthenticateLogicAppsManagementAsync(subscriptionId, accessToken));
+        }
+
+        /// <summary>
+        /// Uses a accessToken to authenticate with Azure.
+        /// </summary>
+        /// <param name="subscriptionId">The ID that identifies the subscription on Azure.</param>
+        /// <param name="accessTokenKey">The token to use to call the Azure management API.</param>
```

Comment:
```suggestion
        /// <param name="accessTokenKey">The secret key to use to fetch access token from the secret provider. This will be used to call the Azure management API.</param>
```